controllers/loadtest_controller.go

```diff
@@ -111,12 +111,12 @@ func (r *LoadTestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		if err = r.Status().Update(ctx, test); err != nil {
 			log.Error(err, "failed to update test status when setting defaults failed")
 		}
-		return ctrl.Result{}, err
+		return ctrl.Result{Requeue: false}, nil
 	}
 	if !reflect.DeepEqual(rawTest, test) {
 		if err = r.Update(ctx, test); err != nil {
 			log.Error(err, "failed to update test with defaults")
-			return ctrl.Result{}, err
+			return ctrl.Result{Requeue: true}, err
```

Comment:
Do you want to requeue this even there is no error? Similar to other ctrl.Result{Requeue: true} below.